evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -92,6 +92,32 @@ def _are_datasets_separated_by_gap(train_index, test_index, gap):
             gap_difference *= test_index.freq
         return index_difference == gap_difference
 
+    def _validate_holdout_datasets(self, X, X_train):
+        """Validate the holdout datasets match out expectations.
+
+        Args:
+            X (pd.DataFrame): Data of shape [n_samples, n_features].
+            objective (Object or string): The objective to use to make predictions.
+            X_train (pd.DataFrame): Training data.
+            y_train (pd.Series): Training labels.
```

Comment:
Docstring doesn't match the function sig!

Reply:
Done!